test/Microsoft.Dnx.TestHost.Tests/TestHostTest.cs

```diff
@@ -21,11 +21,11 @@ public class TestHostTest
 
         public TestHostTest()
         {
-            var libraryManager = PlatformServices.Default.LibraryManager;
-            _testProject = Path.GetDirectoryName(libraryManager.GetLibrary("Sample.Tests").Path);
+            var testProjectFolder = Path.GetDirectoryName(Path.GetDirectoryName(Path.GetDirectoryName(Directory.GetCurrentDirectory())));
```

Comment:
`Path.Combine(Directory.GetCurrentDirectory(), "..", "..", "..")` (or so) would be much more readable.